int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -333,6 +333,18 @@ for (var i = 0; i < buttonSelector.length; i++) {
 		buttonSelector[i].addEventListener("mouseover", edqEmailPhoneValidationCallback);
 	}
 }
+/** Added this function to set the listener tothe button on the selected stage (billing/shipping)
+* For more information see Bug #147798 */
+function setEdqButtonSelector(stageContentLocation) {
+	stageContentLocation = stageContentLocation || "";
+	if (stageContentLocation === "shipping") {
+		edqCurrentSubmitButtonSelector = document.querySelector("[value=submit-shipping]");
```

Comment:
The name `edqCurrentSubmitButtonSelector` in this case is not accurate - the selector is `[value=submit-shipping]`, what you have it as is the `HTMLElement`. You should change the name to reflect this to prevent confusion (also below).

Reply:
The edqCurrentSubmitButtonSelector refers to the current button selector from the touchpoint address/shipping/billing; for this case we're capturing the selector between Shipping and Billing depending on the stage.

Reply:
Sorry, I must have not been clear:

```javascript
edqCurrentSubmitButtonSelector = document.querySelector("[value=submit-shipping]");
```

What I meant was that you have a variable called edqCurrentSubmitButtonSelector, but you're setting it equal to a button. In other words, if we were to try to use `edqCurrentSubmitButtonSelector` again on `document.querySelector` it wouldn't work since it's actually a button, an not a selector.

Reply:
Changed to edqCurrentSubmitButton.